### ISSUE TITLE
ERROR ConnServer GetFile exception: AttributeError: 'NoneType' object has no attribute 'write' in core/src/File/FileRequest.py line 250 > ... > lib/gevent/_ssl3.py line 436

### ISSUE THREAD

**Issue by slrslr** [HelloZeroNet/ZeroNet#2537]
### Step 1: Please describe your environment

  * ZeroNet version: 4478
  * Operating system: W10
  * Web browser: FF
  * Tor status: error
  * Opened port: closed

### Step 2: Describe the problem:

ZeroHello shows the error on the WARNINGS: 1 on the top button , i click to see some error which i click and in the log i see:

ERROR ConnServer GetFile exception: AttributeError: 'NoneType' object has no attribute 'write' in core/src/File/FileRequest.py line 250 > ... > lib/gevent/_ssl3.py line 436

My ZeroHello newsfeed is not loading and ZeroBlog post submission does not respond (button indicate loading, but not saved)

**Comment by HelloZeroNet** [comment]
I think it's not related to the problem, do you see any other error in the log file?

**Comment by slrslr** [comment]
No, i have not seen any other ERRORs around it. I no longer see that error in the log. (there was restart)
